### ISSUE TITLE
deprecation in favor of https://github.com/containers/toolbox

### ISSUE THREAD

**Issue by cgwalters** [coreos/toolbox#60]
We'd like to converge efforts to https://github.com/containers/toolbox

Please comment on e.g. https://github.com/containers/toolbox/pull/302 if you're a maintainer of this project and would like to continue to contribute.

Further, also a good opportunity to ensure you're following the repo, etc.

**Comment by yuqi-zhang** [comment]
I'm definitely +1 on this. Do we plan on supporting existing RHCOS behaviour? Or will we be modifying that to the unified toolbox workflow?

**Comment by miabbott** [comment]
@yuqi-zhang Yeah, we want to retain the existing RHCOS behavior (or something like it).  See the most recent PR that @dustymabe merged into the coreos/toolbox repo - https://github.com/containers/toolbox/pull/308

**Comment by debarshiray** [comment]
[Toolbox](https://github.com/containers/toolbox) now has a reasonable migration path for `coreos/toolbox` users. See: https://github.com/containers/toolbox/issues/689
